src/style.css

```diff
@@ -1 +1,79 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  height: 60px;
+}
+
+.nav__ul {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__li {
+  margin: 0 10px;
+}
+
+.nav__li:nth-child(8) {
+  margin-right: 0;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav__a {
+  color: black;
+  font-size: 12px;
+  text-decoration: none;
+  text-transform: uppercase;
+  padding: 23px 0;
+  display: inline-block;
+}
+
+.nav .is-active {
+  color: #00acdc;
+}
+
+.nav__a:hover {
+  color: #00acdc;
+}
+
+.nav__li:first-child {
+  position: relative;
```

Comment:
add `position: relative;` to all `nav__li`, not only the first one.
it's important because `::after` does not work if we try to apply to another element
<img width="1181" alt="Screenshot 2023-03-27 at 19 53 37" src="https://user-images.githubusercontent.com/32853034/228025826-1184622c-a6e4-4088-aff1-cad3e1618e4b.png">